### ISSUE TITLE
The content settings button should be focused when the modal is hidden

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#4337]
## Description

On a narrow screen, when you close the content settings modal, the trigger (Content settings button) should be focused. Instead,  the body gets the focus.

## Reproduction

1. Go to https://openverse.org/search?q=cat, and narrow your screen.
2. Click on the content settings button inside the search bar.
3. Press <kbd>Tab</kbd>
4. See error: the content setting button is not focused. If you press <kbd>Tab</kbd> again, you'll see that "Skip to content" button is focused, which shows that the body was focused when the modal was hidden.
